llvm/tools/dsymutil/SwiftModule.h

```diff
@@ -0,0 +1,15 @@
+//===- tools/dsymutil/SwiftModule.h ----------------------------*- C++ -*--===//
```

Comment:
Nit: new files shouldn't have the emacs tag anymore: https://discourse.llvm.org/t/is-c-in-header-files-still-relevant/83124
```suggestion
//===- tools/dsymutil/SwiftModule.h ---------------------------------------===//
```